index.html

```diff
@@ -1291,8 +1291,22 @@
             </li>
             <li>Return <em>P</em>.
             </li>
+            <li>If the user agent knows there will be no available presentation
+            displays during the entire user session because of internal or
+            external limitations, then:
+              <ol>
+                <li>
+                  <a>Resolve</a> <em>P</em> with a new <code>PresentationAvailability</code>
+                  object with its <code>value</code> property set to <code>false</code>.
+                </li>
+                <li>
+                  Abort all the remaining steps.
+                </li>
+              </ol>
+            </li>
             <li>If the user agent is unable to continuously <a>monitor the list
-            of available presentation displays</a>, then:
+            of available presentation displays</a> but there might be available
```

Comment:
I don't think being able to monitor later should be added. It might be very possible to not be able to monitor at all but still be able to connect to a device, right?